src/lib/components/grid.tsx

```diff
@@ -1,6 +1,7 @@
 // tslint:disable-next-line:no-unused-variable
 import * as React from 'react';
-import { Component } from 'react';
+import { Component, ReactElement } from 'react';
+import * as ReactDOM from 'react-dom';
```

Comment:
I believe you want this only:

```
import ReactDOM from 'react-dom';
```

There is a default export and you don't need any other named exports it may bring in with `*`.

Reply:
the typescript imports are a little different so the default syntax you wrote here actually fails.